src/main/java/org/fundacionjala/coding/edgar/SortTheInnerContent.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.coding.edgar;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+final class SortTheInnerContent {
+    private SortTheInnerContent() {
+    }
+
+    private static final int FLAG = 3;
+
+    static String sortTheInnerContent(final String words) {
+        String[] arrayWords = words.split(" ");
+        StringBuilder newWord = new StringBuilder();
```

Comment:
You can use a StringJoiner with delimiter blank space.
so the concatenation with an blank space on line 24, 26 and the trim on line 30 will no longer necessary.